build.gradle

```diff
@@ -22,7 +22,7 @@ buildscript {
 
     repositories {
         maven { url 'https://repo.grdev.net/artifactory/kotlin-eap' }
-        maven { url 'https://dl.bintray.com/kotlin/kotlin-dev' }
+        maven { url 'https://repo.grdev.net/artifactory/kotlin-dev' }
```

Comment:
I think we should remove this repository and the kotlin eap repository. I think the artifacts are now on Maven central, and if we need something else again we should add them.

So we should only have the `google()` and `mavenCentral()` repositories.

Reply:
Let's try it.